packages/Icon/src/svg/Arrow-Right-B.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="480" height="480" viewBox="0 0 480 480">
```

Comment:
Can we bring in `ArrowLeftB` too, just for completeness?

Reply:
Actually, I am a little bit confused all icon-related things.

We have `wasabicons` and `@paprika/icon`.

I remember someone mentioned in Slack that `@paprika/icon` shouldn't be consumed directly in Galvanize platform. Is true? Because I see a lot of usages in `workpapers` repo. If it actually shouldn't be consumed directly, should we keep brigades in touch? Can we technically (not just by convention) prevent direct usage of this package?

Due to the fact we should not use `@paprika/icon` in Galvanize apps, should I really add `ArrowLeftB` to this package?

cc: @nahumzs

Reply:
I think is fine to add just what we need to paprika/icons, when the ArrowLeft its need it, we copy/paste it to paprika. 

The reason behind this confusion is that most of our iconset is a paid library, so whenever we want to use and icon in paprika (our open source library) we need to redo it so we don't fall into copyright issues. 

And you are correct we shouldn't use paprika icon on our highbond apps, we should use wasabicons (our paid iconset plus paprika icons mixed). 

At this moment don't have any idea how to prevent consumers to use paprika icon outside of paprika itself.